docs/content/src/learn/tutorials/saas-integrations/sfdc46/import-contacts-into-salesforce-using-ftp/README.md

```diff
@@ -1,4 +1,4 @@
-# Import Contacts Into Salesforce using FTP connector
+# Salesforce & FTP Integration
```

Comment:
I think it's good to say "Imports Contacts..." since it gives a clearer picture of what's happening here.

Reply:
The name was changed here to the template name since this is where we will fetch the template name dynamically from